src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write your code here
+const array = [];
+const spans = document.querySelectorAll('.population');
```

Comment:
I guess there can be a lot of spans on the page, let's specify what spans have you selected